src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -58,15 +72,9 @@
   {:render
    (fn [{:keys [props]}]
      [:div {:style {:textAlign "right"}}
-      [TopNavBarLink {:name "Import Data"
-                      :selected (= (:selected-item props) :import-data)
-                      :onClick (fn [e] ((:on-nav props) :import-data))}]
-      [TopNavBarLink {:name "Workspaces"
-                      :selected (= (:selected-item props) :workspaces)
-                      :onClick (fn [e] ((:on-nav props) :workspaces))}]
-      [TopNavBarLink {:name "Method Repository"
-                      :selected (= (:selected-item props) :methods)
-                      :onClick (fn [e] ((:on-nav props) :methods))}]])})
+      (map (fn [item] [TopNavBarLink {:name (item :name)
+                                      :selected (= (:selected-item props) (item :key))
+                                      :onClick (fn [e] ((:on-nav props) (item :key)))}]) top-nav-bar-items)])})
```

Comment:
Pull `top-nav-bar-items` to a new line so it doesn't get lost in the end. Otherwise good.